pkg/version/version.go

```diff
@@ -16,4 +16,4 @@
 
 package version
 
-const MonitoringAsCode = "1.6.0"
+const MonitoringAsCode = "2.0.0-new-config"
```

Comment:
why not simply 
```suggestion
const MonitoringAsCode = "2.0.0"
```

Reply:
This was just for local debug builds. I would not go with `2.0.0` yet. 
Maybe `2.0.0-next`, since we are going to build next binaries. That
way people don't mistake it for a full release.